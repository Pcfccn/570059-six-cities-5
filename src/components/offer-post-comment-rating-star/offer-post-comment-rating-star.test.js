import React from "react";
import renderer from "react-test-renderer";
import RatingStar from "./offer-post-comment-rating-star";
const noop = () => {};
it(`Should RatingStar render correctly`, () => {
  const tree = renderer
    .create(<RatingStar
      rating={5}
      currentRating={`5`}
      onChange={noop}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
