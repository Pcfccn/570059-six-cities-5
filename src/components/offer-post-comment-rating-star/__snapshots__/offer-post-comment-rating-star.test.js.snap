// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingStar render correctly 1`] = `
Array [
  <input
    checked={true}
    className="form__rating-input visually-hidden"
    id="5-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={5}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="5-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
